solidity/random-beacon/test/RandomBeacon.Relay.test.ts

```diff
@@ -850,10 +766,782 @@ describe("RandomBeacon - Relay", () => {
+                          failedMembersIndices.length
+                        )
+
+                      const currentNotifierBalance = await testToken.balanceOf(
+                        claimSender.address
+                      )
+                      expect(
+                        currentNotifierBalance.sub(initialNotifierBalance)
+                      ).to.be.equal(expectedReward)
+
+                      const currentHeartbeatNotifierRewardsPoolBalance =
+                        await randomBeacon.heartbeatNotifierRewardsPool()
+                      expect(
+                        initialHeartbeatNotifierRewardsPoolBalance.sub(
+                          currentHeartbeatNotifierRewardsPoolBalance
+                        )
+                      ).to.be.equal(expectedReward)
+                    })
+                  }
+
+                  context(
+                    "when there are multiple subsequent failed members indices",
+                    async () => {
+                      await assertNotifyFailedHeartbeatSucceed(
+                        subsequentFailedMembersIndices,
+                        groupThreshold,
+                        (signatures) => signatures,
+                        (signingMembersIndices) => signingMembersIndices
+                      )
+                    }
+                  )
+
+                  context(
+                    "when there is only one failed members index",
+                    async () => {
+                      await assertNotifyFailedHeartbeatSucceed(
+                        [32],
+                        groupThreshold,
+                        (signatures) => signatures,
+                        (signingMembersIndices) => signingMembersIndices
+                      )
+                    }
+                  )
+
+                  context(
+                    "when there are multiple non-subsequent failed members indices",
+                    async () => {
+                      await assertNotifyFailedHeartbeatSucceed(
+                        nonSubsequentFailedMembersIndices,
+                        groupThreshold,
+                        (signatures) => signatures,
+                        (signingMembersIndices) => signingMembersIndices
+                      )
+                    }
+                  )
+
+                  context(
+                    "when there are multiple non-subsequent signing members indices",
+                    async () => {
+                      const newSigningMembersIndices = [
+                        1, 5, 8, 11, 14, 15, 18, 20, 22, 24, 25, 27, 29, 30, 31,
+                        33, 38, 39, 41, 42, 44, 47, 48, 49, 51, 53, 55, 56, 57,
+                        59, 61, 62, 64,
+                      ]
+
+                      const getSignature = (signatures, index) =>
+                        signatures
+                          .slice(2)
+                          .slice(130 * index, 130 * index + 130)
```

Comment:
Can we extract 130 to a local constant and name it? Right now it's a magic number.